osu.Game/Localisation/GameplaySettingsStrings.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class GameplaySettingsStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.GameplaySettings";
+
+        /// <summary>
+        /// "Gameplay"
+        /// </summary>
+        public static LocalisableString GameplaySectionHeader => new TranslatableString(getKey(@"gameplay_section_header"), @"Gameplay");
+
+        /// <summary>
+        /// "General"
+        /// </summary>
+        public static LocalisableString GeneralHeader => new TranslatableString(getKey(@"general_header"), @"General");
+
+        /// <summary>
+        /// "Background dim"
+        /// </summary>
+        public static LocalisableString BackgroundDim => new TranslatableString(getKey(@"dim"), @"Background dim");
+
+        /// <summary>
+        /// "Background blur"
+        /// </summary>
+        public static LocalisableString BackgroundBlur => new TranslatableString(getKey(@"blur"), @"Background blur");
+
+        /// <summary>
+        /// "Lighten playfield during breaks"
+        /// </summary>
+        public static LocalisableString LightenDuringBreaks => new TranslatableString(getKey(@"lighten_during_breaks"), @"Lighten playfield during breaks");
+
+        /// <summary>
+        /// "HUD overlay visibility mode"
+        /// </summary>
+        public static LocalisableString HUDVisibilityMode => new TranslatableString(getKey(@"hud_visibility_mode"), @"HUD overlay visibility mode");
+
+        /// <summary>
+        /// "Show difficulty graph on progress bar"
+        /// </summary>
+        public static LocalisableString ShowProgressGraph => new TranslatableString(getKey(@"show_progress_graph"), @"Show difficulty graph on progress bar");
```

Comment:
This should really be `ShowDifficultyGraph`, no? The graph is a difficulty graph. Not a "progress graph".

Reply:
at the same time in `osu.Game\Overlays\Settings\Sections\Gameplay\GeneralSettings.cs`
```csharp
new SettingsCheckbox
{
    LabelText = GameplaySettingsStrings.ShowProgressGraph,
    Current = config.GetBindable<bool>(OsuSetting.ShowProgressGraph)
},
```
the `OsuSetting.ShowProgressGraph` should get this rename, right?

Reply:
no, the setting rename should not have been done in this pull.